virttest/qemu_vm.py

```diff
@@ -1055,6 +1055,13 @@ def add_memorys(devices, params):
                 backend_options["size_mem"] = "%sM" % params["mem"]
                 if params.get("vm_mem_backend"):
                     backend_options["backend_mem"] = params.get("vm_mem_backend")
+                if params.get("vm_mem_backend") == "memory-backend-file":
+                    if not params.get("vm_mem_backend_path"):
+                        raise virt_vm.VMConfigMissingError(name,
```

Comment:
```suggestion
                        raise virt_vm.VMConfigMissingError(self.name,
```
Since the definition:
```
class VMConfigMissingError(VMError):
...
    def __str__(self):
        return "Missing config '%s' for VM %s" % (self.config, self.name)
```

Reply:
I think I could keep "name" cause line 1053?
name = output[0]

Reply:
It is same in this content, as the `name = self.name` if it was not defined.

Reply:
> I think I could keep "name" cause line 1053? name = output[0]

Could you run the test without the `vm_mem_backend_path` and paste the log about the error of `VMConfigMissingError`? let's check it.

Reply:
In this error, `name` means **VM** name, not the **mem backend** name
```
Missing config 'vm_mem_backend_path' for VM machine_mem
```

Reply:
> It is same in this content, as the `name = self.name` if it was not defined.

As the usage of the `__str__` in `class VMConfigMissingError(VMError)`: `return "Missing config '%s' for VM %s" % (self.config, self.name)`, I think the self.name points to the VM object.
So what is the purpose of calling `name`, not calling the `self.name` directly?

Reply:
Eh~   the initial purpose is call VM's name, and i thought name is same with self.name as:
https://github.com/avocado-framework/avocado-vt/blob/e20d6f47c2198f423bbc8f420b2775e404322416/virttest/qemu_vm.py#L1575

But, you are right, `name` already be overwritten, not vm name any more.
So, right, we need use `self.name`

Reply:
updated